### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by harkue**
fix some typo and camel cases

### CODE REVIEW

File: JSONArray.java [hunk 1]
```diff
@@ -1409,7 +1409,7 @@ public Writer write(Writer writer) throws JSONException {
     public Writer write(Writer writer, int indentFactor, int indent)
             throws JSONException {
         try {
-            boolean commanate = false;
```
Comment (johnjaylward) [inline JSONArray.java]: `needsComma` I think would be a better name than `hasComma`

File: JSONObject.java [hunk 2]
```diff
@@ -2496,7 +2496,7 @@ static final void indent(Writer writer, int indent) throws IOException {
     public Writer write(Writer writer, int indentFactor, int indent)
             throws JSONException {
         try {
-            boolean commanate = false;
```
Comment (johnjaylward) [inline JSONObject.java]: same here, `needsComma` seems like a better fit.

Comment (harkue) [reply JSONObject.java]: ok

**Comment by stleary** [comment]
**What problem does this code solve?**
Refactoring typos from the code - no change to behavior.

**Risks**
Minimal

**Changes to the API?**
None


**Will this require a new release?**
No

**Should the documentation be updated?**
No

**Does it break the unit tests?**
No

**Was any code refactored in this commit?**
var names updated.

**Review status**
**APPROVED**

Starting 3 day comment window

**Pull request merged**